client/components/Cards/MorphologyRelease/index.js

```diff
@@ -0,0 +1,105 @@
+import React, { Fragment } from "react";
+import { WithStore } from "@bbp/nexus-react";
+import TypeIcon from "../../TypeIcon";
+import { find } from "underscore";
+import Perspectivizer from "../../Animations/Perspectivizer";
+import { getProp } from "@libs/utils";
+import InspectLink from "./InspectLink";
+import Contributions from "../Cell/Contributions";
+import { Icon } from "antd";
+import moment from "moment";
+import Download from "../../Download";
+
+const GridResult = ({ value, id }) => {
+  return (
+    <WithStore
+      mapStateToProps={({ types }) => ({
+        hoverType: types.hoverType,
+        types: types.types
+      })}
+      mapDispatchToProps={{}}
+    >
+      {({ hoverType, types }) => {
+        const mostRelevantType = Array.isArray(value["@type"])
```

Comment:
The repetition of `value["@type"]` makes this group of lines hard to read. Also, `typeArray` is not used after declaration, to remove. Proposal:
```
const resourceType = value["@type"];
const mostRelevantType = Array.isArray(resourceType)
  ? resourceType.pop()
  : resourceType;

We can mutate the array since it is not used afterwards.